charts/gardener/controlplane/values-test.yaml

```diff
@@ -0,0 +1,326 @@
+global:
```

Comment:
Instead of coping the entire file, you can set just the required parameters
```yaml
global:
  # Gardener API server configuration values
  apiserver:
    clusterIdentity: garden-cluster-identity
```